template.tpl

```diff
@@ -465,6 +465,14 @@ ___WEB_PERMISSIONS___
               {
                 "type": 1,
                 "string": "https://*.bambuser.com/*"
+              },
+              {
+                "type": 1,
+                "string": "https://lvs-121-prod-eu.web.app/embed.js"
```

Comment:
We actually don't need these since the links we have is 
`https://one-to-one.bambuser.com/embed.js` and `https://one-to-one.bambuser.com/eu/embed.js` which both are ok under the first rule